exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -113,6 +113,9 @@ void invalidConfig() {
     assertThatThrownBy(() -> PrometheusHttpServer.builder().setHost(""))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("host must not be empty");
+    assertThatThrownBy(() -> PrometheusHttpServer.builder().setDefaultAggregationSelector(null))
+        .isInstanceOf(NullPointerException.class)
+        .hasMessage("defaultAggregationSelector");
```

Comment:
Can you add a test on the client side that verifies that exponential histograms are in fact working as expected when we override the default aggregation?